CMakeLists.txt

```diff
@@ -69,6 +69,13 @@ IF(NOT "${Boost_VERSION}" MATCHES "1.53(.*)")
    SET(Boost_LIBRARIES ${BOOST_LIBRARIES_TEMP} ${Boost_LIBRARIES})
 ENDIF()
 
+OPTION( ASSET_BALANCE_SORTING "Keep account_balance objects sorted by asset and amount, to be used in asset_api (ON OR OFF)" ON )
```

Comment:
All code other than CMAKE uses ASSET_BALANCE_SORTED, but CMAKE is looking for ASSET_BALANCE_SORTING. The easier fix is to change it in CMakeLists.txt. 

My opinion: ASSET_BALANCE_SORTING is not as descriptive as ASSET_BALANCE_SORTED. ASSET_BALANCE_SORTING makes me think it is not an ON/OFF option, but perhaps an algorithm choice.

Reply:
Ahhh, Now I see why that was done. CMake doesn't like it when their variables are the same name as compile variables. I think there is a workaround for more recent versions of cmake, but I haven't explored it. I'm cancelling my request for changes.